foundation/settings.py

```diff
@@ -224,11 +240,27 @@ def _parse_email_list(varname):
     }
 
 # Database configuration
+# (optional) Create your local DB
+"""
+CREATE USER okfn WITH PASSWORD 'okfn';
+ALTER ROLE okfn SUPERUSER;
+CREATE DATABASE okfn OWNER okfn;
+"""
 
-db_config = dj_database_url.config(default='sqlite:///development.sqlite3')
 DATABASES = {
-    'default': db_config
+    'default': {
+        'ENGINE': env.get('DB_ENGINE'),
+        'NAME': env.get('DB_NAME'),
+        'USER': env.get('DB_USER'),
+        'PASSWORD': env.get('DB_PASSWORD'),
+        'HOST': env.get('DB_HOST'),
+        'PORT': env.get('DB_PORT'),
+    },
 }
+# If the flag as been set, configure to use proxy
+if env.get("USE_CLOUD_SQL_AUTH_PROXY", None):
```

Comment:
Maybe this should be documented for local development

Reply:
Removed. Copied from google tutorial but we will not use that feature.